build/scripts/find-lint.sh

```diff
@@ -19,7 +19,7 @@ if [ ${1:-""} = "fast" ]
 then args="--fast"
 fi
 
-if [[ -v COMPLEMENT_LINT_CONCURRENCY ]]; then
```

Comment:
The `-v` thing seems to only work on bash 4.2 and above, https://unix.stackexchange.com/a/212192/52320

On macOS 11.1

```
bash --version
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin20)
Copyright (C) 2007 Free Software Foundation, Inc.
```